lib/interfaces/multer-extended-s3-options.interface.ts

```diff
@@ -1,6 +1,10 @@
 import { LoggerService } from '@nestjs/common';
 
 export interface MulterExtendedS3Options {
+  /**
+   * AWS End Point
+   */
+  readonly endpoint?: string;
```

Comment:
As mentioned in the my earlier comment this can be used for digital ocean.
We are using digital ocean spaces for data storage and this was required for us.

This will not be a breaking change, as aws uses computed a default endpoint if one is not specified

Reply:
@semin3276 
Isn't the `endpoint` is actually the address of DigitalOcean space?
Maybe we should separate the property in a different interface for DigitalOcean space config?